profile/README.md

```diff
@@ -3,12 +3,10 @@
 Welcome to the DAI course at [HEIG-VD](https://heig-vd.ch), Switzerland!
 
 ## Getting started
-
```

Comment:
You shouldn't have removed the line between the title and the HTML comment. Maybe it's your IDE that did format the file for you? Please re-add the lines.